.changelog/DEV-1766.yaml

```diff
@@ -0,0 +1,16 @@
+# Type of change
+change_type: bug_fix
+
+# The name of the component
+component: deployment
+
+# (Optional) One or more tracking issues or pull requests related to the change
+issues: []
+
+# A brief description of the change.
+note: "fix multiple deployments sharing the same feature list bug"
```

Comment:
nit: I think it would be useful if the note can be more descriptive of the issue / bug that is fixed.